index.ts

```diff
@@ -71,8 +71,10 @@ async function main() {
                   const additionalLabelsToAdd = projectIssue?.assignee?.login
                     ? ["Unavailable"]
                     : [];
+                    const isUnavailableTag = projectIssue?.labels?.some((item)=>item.name==="Unavailable");
```

Comment:
shouldn't we check devpool issue instead? Partner project issues can't have the `Uavailable` tag

```
const isUnavailableTag = devpoolIssue?.labels?.some((item)=>item.name==="Unavailable");
```

Reply:
Actailly you are right with that !